aospy/calc.py

```diff
@@ -767,3 +768,23 @@ def load(self, dtype_out_time, dtype_out_vert=False, region=False,
         if plot_units:
             data = self.var.to_plot_units(data, dtype_vert=dtype_out_vert)
         return data
+
+def _add_units_and_description_attrs(data, units, description, dtype_out_vert):
```

Comment:
Now that I see this, the function name seems clunky.  How about _add_metadata_as_attrs` instead?  That is less of a mouthful and also leaves the door open to us adding additional attrs within the function in the future, without renaming.